README.md

```diff
@@ -1 +1,106 @@
-# coord-dsl
\ No newline at end of file
+# coord-dsl
+
+## Install
+
+```bash
+pip install coord-dsl
+```
+
+### Editable Install
+
+```bash
+pip install -e .
+```
+
+## FSM
+
+- A finite state machine (FSM) DSL implementation following the [book](https://robmosys.pages.gitlab.kuleuven.be/composable-and-explainable-systems-of-systems.pdf).
+- The FSM should be defined in a `.fsm` file.
```

Comment:
Should probably refer to [rosym-project/coord2b](https://github.com/rosym-project/coord2b) for execution of the C++ generated structs.